Graph Patterns (#324)
Microsoft/api-guidelines: graph/GuidelinesGraph.md

| Pattern                 | Description                                                                                    | Reference                                                            |
|-------------------------|------------------------------------------------------------------------------------------------|----------------------------------------------------------------------|

> | Type Hierarchy          | The ability to model parent-child relationships using subtypes.                                | [Subtypes](./patterns/Modelling%20with%20Subtypes%20Pattern.md) |

Reviewer: ```suggestion
| Type Hierarchy          | The ability to model is-a relationships using subtypes.                                | [Subtypes](./patterns/Modelling%20with%20Subtypes%20Pattern.md) |
```
